lib/modules/datasource/artifactory/__snapshots__/index.spec.ts.snap

```diff
@@ -5,18 +5,22 @@ exports[`modules/datasource/artifactory/index getReleases parses real data (file
   "registryUrl": "https://jfrog.company.com/artifactory",
   "releases": [
     {
+      "registryUrl": "https://jfrog.company.com/artifactory",
```

Comment:
Previously we tried to avoid having result.release.registryUrl defined if it was the same as result.registryUrl.

My ideal approach would be:
- If all results come from one registry then set result.registryUrl
- If results come from a mix of registries then unset result.registryUrl and set every result.release.registryUrl

Does that align with what you're trying to do?